src/cloudai/_core/test_template.py

```diff
@@ -149,7 +149,11 @@ def gen_exec_command(
         """
         if not nodes:
             nodes = []
-        assert self.command_gen_strategy is not None
+        if self.command_gen_strategy is None:
+            raise ValueError(
+                "command_gen_strategy is missing. Ensure the strategy is registered in src/cloudai/__init__.py "
```

Comment:
`src/cloudai/__init__.py` is valid for this repo runner only. In Extensions, for example, registration is done in addition to `__init__.py`. Let's mention `Registry` here and maybe expand documentation on it .

Reply:
Sure, I will do.